scripts/variantstore/scripts/close_bed_file_gaps.py

```diff
@@ -34,11 +36,17 @@
 else:
 	print("We do not have an interval list")
 
+
 last_contig = ""
 last_ending = 0
 interval_size = binsize_kb * 1000
 print(f"Interval size is {interval_size}")
 
+def maybe_say(s):
+	if args.verbose:
+		print(s)
```

Comment:
Preexisting issue, but I would not be at all upset if this entire file got reindented sanely.

Reply:
I guess that file has tabs and the rest have spaces, so we can convert them from tabs to spaces one day